lib/data_update_scripts/20220111192031_insert_apple_connect_broadcast_message.rb

```diff
@@ -0,0 +1,18 @@
+module DataUpdateScripts
+  class InsertAppleConnectBroadcastMessage
+    def run
+      title = "Welcome Notification: apple_connect"
+      return if Broadcast.find_by title: title
+
+      message = "You're on a roll! 🎉  Do you have an Apple account? " \
```

Comment:
this text matches both the other (recent) connect options like forem or facebook, as well as the factory.

Question: should this also go into the seeds.rb? I think so.